include/States.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ALE_STATES_H
+#define ALE_STATES_H
+
+#include<vector>
+#include <gsl/gsl_interp.h>
+ 
+namespace ale {
+
+enum interpolation {
+  /// Interpolate using linear interpolation
+  linear,
```

Comment:
This is definitely a personal preference, but I think it is easier to identify enum values when they are all caps, almost treating them like a constant.  After looking at ISIS, it doesn't seem like we have a standard for defining enums.

Reply:
We define an interpolation enum in ale.h. Should we redefine the enum in states.h or use the same enum from ale.h?

Reply:
@kaitlyndlee I moved this into States because I wasn't sure whether the content in ale.h was going to stay in there or not, so this is a good question.